src/account/account.ts

```diff
@@ -123,7 +123,9 @@ export async function uploadPortableAccount(
   const topic = createCredentialsTopic(username, password)
   const socWriter = connection.bee.makeSOCWriter(privateKey)
 
-  return socWriter.upload(connection.postageBatchId, topic, encryptedBytes)
+  return socWriter.upload(connection.postageBatchId, topic, encryptedBytes, {
+    pin: true,
```

Comment:
why did you set `pin` here? does not help to actually find it from another Bee client